indra/explanation/paths_graph.py

```diff
@@ -287,14 +287,15 @@ def sample_single_path(pg, source, target, signed=False, target_polarity=0,
                 out_edges = pg.out_edges(current_node, data=True)
             else:
                 out_edges = pg.out_edges(current_node)
+            out_edges.sort()
```

Comment:
Is there any way to avoid this sort call? Sampling single paths will get called over and over and having to run a sort in the innermost loop here may have a high cost.

Reply:
Well it has to be regularized some how. Perhaps we can set a parameter to make it optional...? Let me take a look...

Reply:
That would be tough. Perhaps if we had some kind of global setting to see if we're in a test?

Reply:
Not pretty but we could do ```
if 'TRAVIS' in os.environ:
    out_edges.sort
```

Reply:
That wouldn't help when testing locally though.

Reply:
We might be able to introspect into the stack and see if nose is in it.